### PR TITLE
Feat/svc

### PR DESCRIPTION
**Pull request by kilosonc**

### CODE REVIEW

File: src/components/neworedit/Navigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+  return (
+    <PageWithBreadcrumb>
+      <Row>
+        <Col span={16} offset={4}>
+          <Row>
+            <Col span={12}>
+              {
+                isInstancePage
+                  ? <PageTitle>{intl.formatMessage({ id: 'pages.groups.New cluster' })}</PageTitle>
```
Comment (xuzhu-591) [inline src/components/neworedit/Navigation/index.tsx:104]: cluster

File: src/components/neworedit/Navigation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                  : <PageTitle>{intl.formatMessage({ id: 'pages.groups.newapplication' })}</PageTitle>
+              }
+            </Col>
+            <Col span={12} style={{ textAlign: 'right' }}>
+              <Button
+                type="link"
+                onClick={() => history.push(newV1)}
```
Comment (xuzhu-591) [inline src/components/neworedit/Navigation/index.tsx:111]: this button does not work

File: src/components/neworedit/components/BaseInfoForm/index.tsx [hunk 3]
```diff
@@ -28,10 +28,10 @@ interface Props {
 const BaseInfoForm: React.FC<Props> = (props: Props) => {
   const {
     form,
-    appType = AppOrClusterType.GIT_IMPORT,
+    appType = AppOrClusterType.GIT,
```
Comment (xuzhu-591) [inline src/components/neworedit/components/BaseInfoForm/index.tsx:31]: cluster

File: src/components/neworedit/Navigation/index.tsx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+`;
+
+interface ItemCardProps {
+  avatar: React.ReactNode;
+  title: React.ReactNode;
+  description: string;
+  onClick: () => void;
+}
+
+const ItemCard = (props: ItemCardProps) => {
```
Comment (xuzhu-591) [inline src/components/neworedit/Navigation/index.tsx]: ItemCard has been extract to components dir

File: src/locales/en-US/pages.ts [hunk 5]
```diff
@@ -92,7 +94,7 @@ export default {
   'pages.groups.id': 'Group ID',
   'pages.groups.groupsAndApplications': 'Subgroups and applications',
   'pages.groups.New subgroup': 'New subgroup',
-  'pages.groups.New application': 'New application',
+  'pages.groups.newapplication': 'New application',
   'pages.groups.New group': 'New group',
   'pages.groups.New cluster': 'New application instance',
```
Comment (xuzhu-591) [inline src/locales/en-US/pages.ts:101]: cluster

File: src/pages/applications/Instances/v1/index.tsx [hunk 6]
```diff
@@ -269,6 +250,29 @@ export default () => {
     });
   }, [environment, fullPath]);
 
+  const onCopyClick = useCallback(() => {
```
Comment (xuzhu-591) [inline src/pages/applications/Instances/v1/index.tsx:257]: copying of chart type instance does not work

File: src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx [hunk 7]
```diff
@@ -0,0 +1,267 @@
...
+import {
+  useEffect, useMemo, useRef, useState,
+} from 'react';
+import { useRequest } from '@@/plugin-request/request';
+import { useModel } from '@@/plugin-model/useModel';
+import { useHistory } from 'umi';
+import PageWithBreadcrumb from '@/components/PageWithBreadcrumb';
+import styles from '@//pages/applications/NewOrEdit/index.less';
+import HSteps from '@/components/HSteps';
+import BaseInfoForm from '../../../../components/neworedit/components/BaseInfoForm';
```
Comment (xuzhu-591) [inline src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx]: what about `@/components/....` ?

File: src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+  Affix, Button, Col, Form, Row,
+} from 'antd';
+import {
+  useEffect, useMemo, useRef, useState,
+} from 'react';
+import { useRequest } from '@@/plugin-request/request';
+import { useModel } from '@@/plugin-model/useModel';
+import { useHistory } from 'umi';
+import PageWithBreadcrumb from '@/components/PageWithBreadcrumb';
+import styles from '@//pages/applications/NewOrEdit/index.less';
```
Comment (xuzhu-591) [inline src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx]: what is `@//`

File: src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+      title: intl.formatMessage({ id: 'pages.newV2.step.audit' }),
+      disabled: !currentStepIsValid(2),
+    },
+  ].filter((_, index) => pageOrders.includes(index));
+
+  const onCurrentChange = (cur: number) => {
+    setCurrent(cur);
+  };
+  const prev = () => {
+    setCurrent(current - 1);
```
Comment (xuzhu-591) [inline src/pages/applications/NewOrEdit/v2/ChartDeploy.tsx]: how does it work when page indexes is `[0, 2, 3]` ?

File: src/pages/applications/NewOrEdit/v2/ImageDeploy.tsx [hunk 10]
```diff
@@ -9,16 +9,17 @@ import { useHistory } from 'umi';
 import PageWithBreadcrumb from '@/components/PageWithBreadcrumb';
 import styles from '@//pages/applications/NewOrEdit/index.less';
 import HSteps from '@/components/HSteps';
-import BaseInfoForm from '../components/BaseInfoForm';
+import BaseInfoForm from '../../../../components/neworedit/components/BaseInfoForm';
```
Comment (xuzhu-591) [inline src/pages/applications/NewOrEdit/v2/ImageDeploy.tsx]: @/components

File: src/pages/instances/Pods/v2/ButtonBar.tsx [hunk 11]
```diff
@@ -53,7 +53,13 @@ function ButtonBar(props: ButtonBarProps) {
         successAlert(intl.formatMessage({ id: 'pages.message.cluster.rollback.hint' }));
         break;
       case 'editCluster':
-        history.push(cluster?.git?.url ? `/instances${fullPath}/-/editv2/gitimport` : `/instances${fullPath}/-/editv2/imagedeploy`);
+        if (cluster?.git?.url) {
+          history.push(`/instances${fullPath}/-/editv2/git`);
+        } else if (cluster?.image) {
+          history.push(`/instances${fullPath}/-/editv2/image`);
+        } else {
+          history.push(`instances${fullPath}/-/editv2/image`);
```
Comment (xuzhu-591) [inline src/pages/instances/Pods/v2/ButtonBar.tsx]: chart?

File: src/pages/templates/Components/Form.tsx [hunk 12]
```diff
@@ -12,6 +12,19 @@ export const TemplateForm = (props: { editRepository?: boolean, onRepositoryBlur
 
   return (
     <div>
+      <Form.Item
```
Comment (xuzhu-591) [inline src/pages/templates/Components/Form.tsx:15]: required

**Review by bysph — APPROVED** [review]

**Pull request merged**